src/Assert.php

```diff
@@ -1810,6 +1943,12 @@ public static function isNonEmptyList($array, $message = '')
     }
 
     /**
+     * @psalm-pure
+     * @psalm-template T
+     * @psalm-param mixed|array<T> $array
+     * @psalm-assert array<string, T> $array
+     * @psalm-assert !empty $array
```

Comment:
`isMap` allows for empty arrays.

Reply:
Oh yep, that changed recently indeed!

Reply:
f47f113123cee9bcf18ed073c601839fec8045fb